tools/breaking-changes/generate-methods-props.ts

```diff
@@ -53,13 +54,37 @@ function getSchematicsData(breakingChange: any): any {
   schematicsData.class = breakingChange.apiElementName;
   schematicsData.importPath = breakingChange.entryPoint;
   schematicsData.deprecatedNode = breakingChange.deletedMember.name;
-  schematicsData.comment = `${breakingChange.deletedComment} ${breakingChange.migrationComment}`;
+  schematicsData.comment = getShematicsComment(breakingChange);
```

Comment:
```suggestion
schematicsData.comment = getSchematicsComment(breakingChange);
```

very minor --> getSch*

Reply:
fixed